### PR TITLE
feat!: Update for SE2 release

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

File: src/admin.ts [hunk 1]
```diff
@@ -1,9 +1,5 @@
 import axios from "axios";
-import {
-  Runnable,
-  VersionedRunnable,
-  UserFunctionsParams,
-} from "./types/runnable";
+import { Module, VersionedModule, UserFunctionsParams } from "./types/module";
```
Comment (MacklinEngineering) [inline src/admin.ts:2]: `AvailableFunction` needs to change `fqfn` and `fqfnURI` below.

Comment (MacklinEngineering) [reply src/admin.ts:2]: Did the smoke tests pass on your machine? Otherwise, code looks good, other than the small above change :)

**Comment by MacklinEngineering** [comment]
Did the smoke tests pass on your machine? Otherwise, code looks good, other than the small above change :)

**Review by MacklinEngineering — APPROVED** [review]

**Comment by ospencer** [comment]
@MacklinEngineering yes! Tests are all passing.

**Pull request merged**